packages/quantic/force-app/main/default/lwc/quanticCitation/quanticCitation.js

```diff
@@ -115,7 +115,7 @@ export default class QuanticCitation extends NavigationMixin(LightningElement) {
 
     this.tooltipIsDisplayed = false;
     this.shouldShowTooltipAfterDelay = false;
-    this.tooltipComponent.hideTooltip();
+    // this.tooltipComponent.hideTooltip();
```

Comment:
this is just to make it easier to click on the citation button for the sake of the POC. This will be fixed in another PR (the issue where the tooltip disappears)